third_party/terraform/tests/resource_spanner_database_test.go

```diff
@@ -26,6 +26,18 @@ func TestAccSpannerDatabase_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet("google_spanner_database.basic", "state"),
 				),
 			},
+			{
+				Config: testAccSpannerDatabase_basicUpdate(instanceName, databaseName),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
I'm not sure this check adds any value to this test. However, it _is_ probably worth adding an import step after each config step, rather than just at the end.